Source/Prism/Logging/DebugLogger.cs

```diff
@@ -5,6 +5,7 @@
 
 namespace Prism.Logging
 {
+#if DEBUG
```

Comment:
Remove this check.  This will break anyone using this class as Prism is not released in Debug mode, so this class will now be missing from the assembly.